src/RewardsStreamerMP.sol

```diff
@@ -236,19 +258,35 @@ contract RewardsStreamerMP is ReentrancyGuard {
 
         bool success = REWARD_TOKEN.transfer(to, amount);
         if (!success) {
-            revert StakingManager__TransferFailed();
+            revert StakeManager__TransferFailed();
         }
     }
 
-    function getStakedBalance(address userAddress) external view returns (uint256) {
-        return users[userAddress].stakedBalance;
+    function calculateMP(uint256 _balance, uint256 _deltaTime) public pure returns (uint256) {
```

Comment:
given that it's used only in one point, doesn't it makes sense to have it internal or hardcoded like before?
I don't see a reason to have a function like that called externally, unless you call it for a specific user

Reply:
It's important for UI and tests. we can have an internal that is called by this public function (and make it external) for gas sake.